pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -129,12 +129,13 @@ func getConfigurationName(args []string) (string, error) {
 
 //  Get the configurator based on the ipfailover type.
 func getPlugin(name string, f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions) (ipfailover.IPFailoverConfiguratorPlugin, error) {
-	//  Currently, the only supported plugin is keepalived (default).
-	plugin, err := keepalived.NewIPFailoverConfiguratorPlugin(name, f, options)
+	var plugin ipfailover.IPFailoverConfiguratorPlugin
+	var err error
 
 	switch options.Type {
 	case ipfailover.DefaultType:
-		//  Default.
+		//  Currently, the only supported plugin is keepalived (default).
+		plugin, err = keepalived.NewIPFailoverConfiguratorPlugin(name, f, options)
```

Comment:
If we are cleaning up here, why not just do an if check here for error and return `plugin, fmt.Errorf(...)` here? 
Basically move code from line numbers 144-148 in our code up here. And whilst we are at it, it maybe a better idea to remove the switch/case and convert to an if statement instead (since its only 1 type and we haven't had the need to add more types here). And so the default case `return nil, fmt.Errorf("No plugins available to handle type %q", options.Type)` just becomes the last line in this function? That should make it cleaner and have an easier flow to understand.